### ISSUE TITLE
BUG - Umezawa's Jitte; No triggers when equipped creature dies in combat.

### ISSUE THREAD

**Issue by Chewbaklava** [magefree/mage#9271]
Situation:

Umezawa's Jitte is equipped to an attacking 1/1. The attacker is blocked by defending player's 1/1. Both creatures die.

The Jitte should trigger and get +2 counters. However, this doesn't happen. Pls Fix.

<3
-Chewbaklava

**Comment by PurpleCrowbar** [comment]
Duplicate of #9129
